Testing/SystemTests/tests/analysis/PearlPowderDiffractionScriptTest.py

```diff
@@ -23,10 +23,19 @@ def requiredFiles(self):
         filenames.extend(('PEARL/Focus_Test/Calibration/pearl_group_12_1_TT70.cal',
                           'PEARL/Focus_Test/Calibration/pearl_offset_15_3.cal',
                           'PEARL/Focus_Test/Calibration/van_spline_TT70_cycle_15_4.nxs',
-                          'PEARL/Focus_Test/Attentuation/PRL112_DC25_10MM_FF.OUT'))
+                          'PEARL/Focus_Test/Attentuation/PRL112_DC25_10MM_FF.OUT',
+                          # reference files
+                          'PEARL/Focus_Test/RefFiles/Ref_PRL92476_92479.nxs',
+                          'PEARL/Focus_Test/RefFiles/Ref_PRL92476_92479-0.gss',
+                          'PEARL/Focus_Test/RefFiles/Ref_PRL92476_92479_d_xye-0.dat',
+                          'PEARL/Focus_Test/RefFiles/Ref_PRL92476_92479_tof_xye-0.dat'
+                          ))
```

Comment:
Good. There is a `reference` subdirectory where many system tests put their reference files in a flat structure. But I think this is better as this test has several files and also the focus part of the test and the calibration part of the test.